dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -10,335 +10,98 @@ nav_section: prebid-amp
 
 <div class="bs-docs-section" markdown="1">
 
-# Show Prebid Ads on AMP Pages (Alpha)
+# Show Prebid Ads on AMP Pages
 {: .no_toc}
 
-{% include amp-deprecation-warning.md %}
-
 This page has instructions for showing ads on AMP pages using Prebid.js.
 
-These instructions assume you have read [How Prebid on AMP Works]({{site.github.url}}/dev-docs/how-prebid-on-amp-works.html).
+At a high level, our implementation uses [Prebid Server][PBS] to talk to [AMP ads][AMPads] using the [AMP Real Time Config (RTC)][RTC-Overview] protocol.
```

Comment:
The "AMP ads" link directs users to "https://github.com/ampproject/amphtml/blob/master/ads/google/a4a/docs/a4a-readme.md".  I'm not sure this is where we want to direct users.  Might be better to say "our implementation uses Prebid Server to talk to the AMP runtime using..."